todo_app.sql

```diff
@@ -0,0 +1,88 @@
+-- 1
+DROP TABLE IF EXISTS tasks;
+
+-- 2, 3
+CREATE TABLE tasks (
+    id SERIAL PRIMARY KEY,
+    title varchar(255) NOT NULL,
+    description text,
+    created_at timestamp NOT NULL,
+    updated_at timestamp NOT NULL,
+    completed boolean NOT NULL NOT NULL
+);
+
+-- 4, 5
+ALTER TABLE tasks
+DROP COLUMN completed;
+
+-- 6
+ALTER TABLE tasks
+ADD COLUMN completed_at timestamp;
+
+ALTER TABLE tasks
+ALTER COLUMN completed_at SET DEFAULT NULL;
+
+-- 7
+ALTER TABLE tasks
+ALTER COLUMN updated_at SET NOT NULL;
+
+ALTER TABLE tasks
+ALTER COLUMN updated_at SET DEFAULT now();
+
+-- 8
+INSERT INTO tasks
+VALUES (DEFAULT, 'Study SQL', 'Complete this exercise', now(), now(), NULL);
+
+-- 9
+INSERT INTO tasks
+VALUES (DEFAULT, 'Study PostgreSQL', 'Read all the documentation', now(), now(), NULL);
```

Comment:
This should be only inserting two pieces of data. Call out the columns you want to insert into so that you don't need to specify DEFAULT, and NULL for the other columns.